Makefile

```diff
@@ -52,6 +53,10 @@ else
 REL_OSARCH=linux/$(OSARCH)
 endif
 
+# Run make images DOCKER_PLATFORMS="linux/amd64,linux/arm64" BUILDX_OUTPUT_TYPE=registry IMAGE_PREFIX=[yourregistry]/vc to push multi-platform
```

Comment:
@hwdef I guess this should also include some info for guys using windows? Could you help some?

Reply:
ok, I will modify this line.